app/views/posts/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1> Posts list </h1>
\ No newline at end of file
+<h1> Posts list </h1>
+<%= render partial: "shared/usercard", locals: { user: @user} %>
+<% @posts.each do |post|%>
+  <%= render partial: "shared/postcard", locals: { post: post} %>
+  <%= render partial: "shared/commentscard", locals: {comments: @comments.where(post_id: post.id).limit(5)}%>
+<% end %>
```

Comment:
- [ ] Please edit the post index view design to be like the given design and kindly ensure that you add all elements in the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/180583205-87516727-01d5-49f8-9177-5fff7a05edf6.png)  | ![image](https://user-images.githubusercontent.com/68796232/180583229-f86bde3b-559c-4d69-89aa-a4f0a3d1fb23.png)  |